oem/ibm/libpldmresponder/utils.hpp

```diff
@@ -29,13 +28,11 @@ struct CustomFD
     CustomFD(CustomFD&&) = delete;
     CustomFD& operator=(CustomFD&&) = delete;
 
-    CustomFD(int fd, bool closeOnOutScope = true) :
-        fd(fd), closeOnOutScope(closeOnOutScope)
-    {}
+    CustomFD(int fd) : fd(fd) {}
```

Comment:
what happens to the fd in async case? do we not need the outOfScope check?

Reply:
this is the extra CustomFD class in async case which was not used but in sync case it's required to close when object out of scope.
in async case, DMA class handles both file descriptor